stable/neo4j/templates/core-statefulset.yaml

```diff
@@ -1,10 +1,17 @@
-apiVersion: "apps/v1beta1"
+apiVersion: "apps/v1beta2"
 kind: StatefulSet
 metadata:
   name: "{{ template "neo4j.core.fullname" . }}"
 spec:
   serviceName: {{ template "neo4j.fullname" . }}
   replicas: {{ .Values.core.numberOfServers }}
+  selector:
+    matchLabels:
```

Comment:
You don't need `chart` and `heritage` here.

Reply:
have updated